domaintools/api.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, username, key, https=True, verify_ssl=True, rate_limit=True,
             raise Exception("The DomainTools API endpoints no longer support http traffic. Please make sure https=True.")
         if proxy_url:
             if isinstance(proxy_url, str):
-                self.extra_request_params['proxies'] = {'http': proxy_url, 'https': proxy_url}
+                    self.extra_request_params['proxies'] = {'http://': proxy_url, 'https://': proxy_url}
```

Comment:
I might be mis-reading the web GUI here but is this an extra tab on this line that doesn't need to be there?